snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -407,6 +417,10 @@ func (b *Bootstrapper) startSyncing(ctx context.Context, acceptedBlockIDs []ids.
 	b.initiallyFetched = b.tree.Len()
 	b.startTime = time.Now()
 
+	// Add the first sample to the EtaTracker to establish an accurate baseline
+	// This ensures t.lowestSample is set correctly from the start
+	b.etaTracker.AddSample(b.initiallyFetched, b.tipHeight-b.startingHeight, b.startTime)
```

Comment:
Just wanted to note that `startSyncing` is called multiple times, so this sample can be added multiple times.

It probably isn't a problem... but wanted to write that down.